sunpy/map/maputils.py

```diff
@@ -1,7 +1,309 @@
+    coordinate object, or through default meta that can be then passed into `Map` 
+    to create a Map object.
+
+    `map_helper.acceptable_meta_keywords()` prints the accepted meta that is used in 
+    creating a sunpy.map.Map
+
+    `map_helper.make_header(data)` will create a map header for the numpy.ndarray data, 
+    either with default meta keywords, or a SkyCoord or frame can be passed to extract
+    reference meta
+
+    Examples
+    --------
+    Printing meta keywords:
+
+        * print the accepeted meta keywords that are used in a sunpy.map.Map header
+          >>> from sunpy import map
+          >>> map.MapHelper.acceptable_meta_keywords() 
+          cunit1  :  Units of the coordinate increments along naxis1 e.g. arcsec **required
+          ...
+          CD2_2  :  Matrix element CDi_j describing the rotation required to align solar North with the top of the image.
+
+        * check is a meta keyword is acceptable and print information about it
+          >>> map.MapHelper.acceptable_meta_keywords('exptime')
+          exptime is an accepted keyword  :  exposure time of observation, in seconds e.g 2
+
+          >>> map.MapHelper.acceptable_meta_keywords('keyword_test')
+          keyword_test  is not in the accepted meta data required for sunpy.map.Map 
+    
+    Creating headers from map_helper:
+
+        * Create a default header with meta data assumed to be Earth-based observer at current time and arcsec units
+          >>> from sunpy import map
+          >>> from sunpy.coordinates import frames
+          >>> from astropy.coordinates import SkyCoord
+          >>> from astropy import units as u
+          >>> import numpy as np
+
+          >>> data = np.random.rand(1024, 1024)
+          >>> header = map.MapHelper.make_header(data)
+          >>> my_map = map.Map(data, header)
+        
+        * Create a header from a reference SkyCoord object
+          >>> my_coord = SkyCoord(100*u.arcsec, 100*u.arcsec, obstime = '2013-10-28 01:00', frame = frames.Helioprojective)
+          >>> header = map.MapHelper.make_header(data, my_coord)
+          >>> my_map = map.Map(data, header)
+
+
+
+    """
+    def acceptable_meta_keywords(self, *key):
+        """
+        prints the accepted metadata keywords that are used when
+        creating a `sunpy.map.Map`.
+        """
+        if key:
+            for k in key:
+                if k in acceptable_meta.keys():
+                    print(k, ' : ', acceptable_meta[k])
+                else:
+                    print(k, ' is not in the accepted meta data required for sunpy.map.Map \n The accepted keywords are : \n', list(acceptable_meta.keys()))
+            return
+
+        for k in acceptable_meta:
+            print(k, ' : ', acceptable_meta[k])
+
+    def make_header(self, data, *coordinate, crval=None, crpix=None, cdelt=None, **kwargs):
+        """
+        Function to create a MetaDict from a coordinate object (SkyCoord or coordinate frame)
```

Comment:
```suggestion
        Function to create a `sunpy.util.meta.MetaDict` from a coordinate object (`astropy.coordinates.SkyCoord` or coordinate frame)
```

Have to check those paths first.